doc/whats_new.rst

```diff
@@ -111,6 +111,8 @@ BUG
 
     - Fix bug in source normal adjustment that occurred when 1) patch information is available (e.g., when distances have been calculated) and 2) points are excluded from the source space (by inner skull distance) by `Eric Larson`_
 
+    - Change default scoring method of :func:`mne.decoding.GeneralizationAcrossTime` and :func:`mne.decoding.TimeDecoding` to estimate the scores within the cross-validation as in scikit-learn. The method can be changed with the ``score_mode`` parameter by `Jean-Remi King`_
```

Comment:
Please put `.. include:: links.inc` at the top of the file, then you can do this to get sklearn link:
```
... as in scikit-learn_. The ...
```